exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/filter/RuntimeFilterRecordBatch.java

```diff
@@ -53,27 +53,27 @@
  * the Query's performance ,but just do a memory transfer by the later RemovingRecordBatch op.
  */
 public class RuntimeFilterRecordBatch extends AbstractSingleRecordBatch<RuntimeFilterPOP> {
-  private SelectionVector2 sv2;
+  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(RuntimeFilterRecordBatch.class);
```

Comment:
Imports...

Reply:
I wonder, rather than fixing these each time I mark a logger as private, should someone run a script to fix up all the files?